desc/objectives/linear_objectives.py

```diff
@@ -2611,6 +2614,118 @@ def __init__(
         )
 
 
+class FixCoilCurrent(FixParameters):
+    """Fixes current(s) in a Coil or CoilSet.
+
+    Parameters
+    ----------
+    coil : Coil
+        Coil(s) that will be optimized to satisfy the Objective.
+    target : nested list of bool, optional
```

Comment:
going to be annoying again, so this should not be a bool right? it should be either the desired current, or None in which case it will default to the coils current?

Reply:
Correct, thanks for catching this. `target` gets passed through to `FixParameters` so it should be the same pytree structure as `coil.params_dict`. I copied the docstring from that parent class